src/applications/disability-benefits/all-claims/pages/ancillaryFormsWizardIntro.js

```diff
@@ -0,0 +1,18 @@
+import { ancillaryFormsWizardDescription } from '../content/ancillaryFormsWizardIntro';
+
+export const uiSchema = {
+  'ui:title': 'Additional disability benefits',
+  'view:ancillaryFormsWizardIntro': {
```

Comment:
Is this property needed? Could we put the `ui:description` on the `uiSchema` directly, like you did with the title?

Reply:
Yeah, good call. I forgot that was a thing until later and didn't think to come back to this. :+1: